canvas_sdk/questionnaires/utils.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+from typing import Any, TypedDict
+
+import yaml
+
+from canvas_sdk.utils.plugins import plugin_only
+
+
+class Response(TypedDict):
+    """A Response of a Questionnaire."""
+
+    name: str
+    code: str
+    code_description: str
+    value: str
+
+
+class Question(TypedDict):
+    """A Question of a Questionnaire."""
+
+    name: str
+    code_system: str
+    code: str
+    code_description: str
+    content: str
+    responses_code_system: str
+    responses_type: str
+    use_in_shx: bool
```

Comment:
Can we rename this (and the similar one at the questionnaire level) to `use_in_social_history` or something even more clear? I don't think it's obvious how it's used based on its naming. @kristenoneill any opinions here?